usr/share/rear/finalize/Linux-i386/670_run_efibootmgr.sh

```diff
@@ -76,6 +76,29 @@ local bootloader partition_block_device partition_number disk efipart
 # EFI\fedora\shim.efi
 bootloader=$( echo $UEFI_BOOTLOADER | cut -d"/" -f4- | sed -e 's;/;\\;g' )
 
+# On a system with software RAID, it is not easily possible to determine the
+# underlying disks automatically. The following makes use of GRUB2_INSTALL_DEVICES
+# to determine the appropriate devices for calling efibootmgr, this requires for
+# example to add the following configuration:
+# GRUB2_INSTALL_DEVICES="/dev/nvme0n1 /dev/nvme0n2"
+# See also https://github.com/rear/rear/issues/3459
+if test "$GRUB2_INSTALL_DEVICES" ; then
+    if [[ $boot_efi_parts == "/dev/md"*"p"* ]]; then
```

Comment:
No - because wow it also falsely matches
e.g. '/dev/mdp' or '/dev/mdfoopbar'.

I think
```
[[ $boot_efi_parts == /dev/md+([0-9])p+([0-9]) ]]
```
should (hopefully) work as intended
but those bash pattern matching syntax rules
always confuse me when patterns get more complicated.

I think normally the string to the right of the `==` operator
does not need any quoting (unless special pattern matching
characters should match literally) or escaping (unless the
pattern string contains spaces).